packages/calcite-components/src/components/segmented-control/segmented-control.tsx

```diff
@@ -341,13 +341,13 @@ export class SegmentedControl
     }
   }
 
-  private handleDefaultSlotChange(event: Event): void {
+  private async handleDefaultSlotChange(event: Event): Promise<void> {
     const items = slotChangeGetAssignedElements(event).filter(
       (el): el is SegmentedControlItem["el"] => el.matches("calcite-segmented-control-item"),
     );
 
+    // await Promise.all(items.map((item) => item.componentOnReady()));
```

Comment:
Nitpick: should this be removed or uncommented?

Reply:
Restored. https://github.com/Esri/calcite-design-system/pull/10818#discussion_r1851012401.